scss/_card.scss

```diff
@@ -12,6 +12,7 @@ $card-border-radius-inner: ($card-border-radius - $card-border-width);
 $card-cap-bg: #f5f5f5;
 
 .card {
+  background-color: #fff;
```

Comment:
Placement of this property is incorrect—please place it before `border`. See the property order in our Sass lint config for more details.